mrp_analytic/views/mrp_view.xml

```diff
@@ -2,18 +2,6 @@
 <openerp>
     <data>
 
-        <record id="mrp_production_tree_view" model="ir.ui.view">
-            <field name="name">mrp.production.tree</field>
-            <field name="model">mrp.production</field>
-            <field name="inherit_id" ref="mrp.mrp_production_tree_view"/>
-            <field name="arch" type="xml">
-                <xpath expr='//field[@name="product_uom"]' position="after">
```

Comment:
why are you removing this file? you can't port it easily: `product_uom` -> `product_uom_id`